pkg/uptimerobot_test.go

```diff
@@ -181,6 +204,34 @@ func TestGetMonitors(t *testing.T) {
 	}
 }
 
+func TestGetMonitorsPages(t *testing.T) {
+	const (
+		baseName    = "monitor"
+		numMonitors = 100
+	)
+	want := make([]string, numMonitors)
```

Comment:
It's not clear to me why that would be better. Can you explain?

Reply:
Instead of generating and holding in memory a slice of 100 strings, we really only need to compare one string at a time. Instead of generating all the strings in a loop, we can generate one just when we need it; this is more efficient on memory.